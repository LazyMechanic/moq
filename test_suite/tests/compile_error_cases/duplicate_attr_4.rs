#[moq::automock]
trait Trait {
    #[moq(default = 1)]
    #[moq(default = 2)]
    const CONST: i32;
}

fn main() {}
