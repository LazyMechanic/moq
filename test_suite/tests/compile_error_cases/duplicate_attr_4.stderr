error: duplicate moq attribute: `default`
 --> tests/compile_error_cases/duplicate_attr_4.rs:4:11
  |
4 |     #[moq(default = 2)]
  |           ^^^^^^^
